### ISSUE TITLE
Trying to update read-only source calendar

### ISSUE THREAD

**Issue by MvS12345** [phw198/OutlookGoogleCalendarSync#643]
**OGCS Version**: v2.7.0.0 
**Installed or Portable**: Installed

**Problem Description**  
I am  syncing from Google to Outlook. The Google calendar is from another user and I have read-only access. OGCS syncs OK, but takes several minutes to sync around 50 events. The log shows mutiple repeats of the following:
```
2018-12-18 09:53:11,655 DEBUG  12 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [0] -  This error seems to be a new transient issue, so treating it with exponential backoff...
2018-12-18 09:53:11,655 WARN  12 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [0] -  API rate limit reached. Backing off 1sec before retry.
2018-12-18 09:53:12,854 WARN  12 OutlookGoogleCalendarSync.GoogleOgcs.Calendar [0] -  Google.Apis.Requests.RequestError
Forbidden [403]
Errors [
	Message[Forbidden] Location[ - ] Reason[forbidden] Domain[global]
```

**Steps to Reproduce the Issue**  
Above happens each time I sync.
Interval is set to 4 hours, and date range from -30 to +365

**Comment by phw198** [comment]
:arrow_double_up: Please check if you can replicate this error on the latest ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg) by going to the `About` tab in OGCS, ticking the `Check for Alpha releases` box and then clicking `Check for Update`. 

If you are on version v2.7.3 or earlier, due to a [GitHub change](https://github.com/phw198/OutlookGoogleCalendarSync/issues/497) you will need to [download the latest version directly from the project website](https://phw198.github.io/OutlookGoogleCalendarSync/).

**Comment by MvS12345** [comment]
Yes issue remains after updating to 2.7.7
Full log file attached. I also used new auto error reprting feature to send you 50 lines of log

**Comment by phw198** [comment]
The logfile you uploaded still shows you using v2.7.0. However, I think this is still a bug in the latest alpha, 

:package: Please could you see if [hotfix v2.7.7.2.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/2701302/v2.7.7.2.zip) works better ([see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).

Let me know how it goes!

**Comment by MvS12345** [comment]
Now runs in about 2 minutes – which seems to be a substantial improvement.

New log file attached

**Comment by phw198** [comment]
Great, thanks for confirming 👍 
The possible longer run time you are seeing is likely due to issue #599.

**Comment by MvS12345** [comment]
Thanks Paul,
 I'll  keep checking both issues on github, in case there are any further improvements 
I'll delete my log files from github as they can be viewed by all, and belong to someone else's calendar

**Comment by phw198** [comment]
Reopening this until the fix is released in the next alpha build v2.7.8
